test/Tests/src/Data/Numbers_Spec.enso

```diff
@@ -615,76 +676,104 @@ spec =
             -3098 . round -3 . should_equal -3000
             -3101 . round -3 . should_equal -3000
 
-        Test.specify "Banker's rounding handles half-way values correctly" <|
-            12350 . round -2 use_bankers=True . should_equal 12400
-            12250 . round -2 use_bankers=True . should_equal 12200
-            -12350 . round -2 use_bankers=True . should_equal -12400
-            -12250 . round -2 use_bankers=True . should_equal -12200
-
-        Test.specify "Banker's rounding handles non-half-way values just like normal rounding" <|
+        Test.specify "Can round negative integers to a specified number of negative places with banker's rounding correctly" <|
+            12300 . round -2 use_bankers=True . should_equal 12300
+            12301 . round -2 use_bankers=True . should_equal 12300
             12330 . round -2 use_bankers=True . should_equal 12300
+            12349 . round -2 use_bankers=True . should_equal 12300
+            12350 . round -2 use_bankers=True . should_equal 12400
+            12351 . round -2 use_bankers=True . should_equal 12400
             12370 . round -2 use_bankers=True . should_equal 12400
             12430 . round -2 use_bankers=True . should_equal 12400
             12470 . round -2 use_bankers=True . should_equal 12500
 
+            12249 . round -2 use_bankers=True . should_equal 12200
+            12250 . round -2 use_bankers=True . should_equal 12200
+            12251 . round -2 use_bankers=True . should_equal 12300
+
+            -12300 . round -2 use_bankers=True . should_equal -12300
+            -12301 . round -2 use_bankers=True . should_equal -12300
             -12330 . round -2 use_bankers=True . should_equal -12300
+            -12349 . round -2 use_bankers=True . should_equal -12300
+            -12350 . round -2 use_bankers=True . should_equal -12400
+            -12351 . round -2 use_bankers=True . should_equal -12400
             -12370 . round -2 use_bankers=True . should_equal -12400
             -12430 . round -2 use_bankers=True . should_equal -12400
             -12470 . round -2 use_bankers=True . should_equal -12500
 
+            -12249 . round -2 use_bankers=True . should_equal -12200
+            -12250 . round -2 use_bankers=True . should_equal -12200
+            -12251 . round -2 use_bankers=True . should_equal -12300
+
         Test.specify "Returns the correct type" <|
             231 . round 1 . should_be_a Integer
             231 . round 0 . should_be_a Integer
             231 . round . should_be_a Integer
             231 . round -1 . should_be_a Integer
 
-        Test.specify "Decimal places out of range" <|
-            3 . round 16 . should_fail_with Illegal_Argument
-            3 . round -16 . should_fail_with Illegal_Argument
-
         Test.specify "Input out of range" <|
-            100000000000000 . round . should_fail_with Illegal_Argument
-            -100000000000000 . round . should_fail_with Illegal_Argument
```

Comment:
I see these 'negative' tests removed, but I see no 'positive' tests for these cases taking their place. We stop testing what happens on `3 . round 16`. and `100000000000000 . round`. That does not seem right, I think?

I think we should add these back as positive cases showing the new behaviour.

Unless there is something preventing us from doing so?

Reply:
I took these out because Integer.round now returns its argument immediately without checking for bad input, which was suggested to me in an earlier review. Do we prefer to do that?

Reply:
Done.